packages/utilities/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
refactor config should be the same with other package tsconfig

Reply:
noted sir